File: Modules/Project/Milit-Panchasara/Uber-APIs/UberAPI/Models/RideType.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace UberAPI.Models
{
    public partial class RideType
    {
        public RideType()
        {
            Trips = new HashSet<Trip>();
            Vehicles = new HashSet<Vehicle>();
        }

        public long RideTypeId { get; set; }
        public string RideName { get; set; }
        public double PricePerKm { get; set; }
        public int SeatingCapacity { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? ModifiedAt { get; set; }

        public virtual ICollection<Trip> Trips { get; set; }
        public virtual ICollection<Vehicle> Vehicles { get; set; }
    }
}

File: Modules/Project/Milit-Panchasara/Uber-Angular/src/app/models/RideType.ts
export interface RideTypeInterface{
    rideTypeId?: number,
    rideName: string,
    pricePerKm: number,
    seatingCapacity: number,
    createdAt?: string,
    modifiedAt?: string
}

export class RideType implements RideTypeInterface {
    rideTypeId = null;
    rideName: '';
    pricePerKm: 0;
    seatingCapacity: 0;
    createdAt: null;
    ModifiedAt: null;
}
